install/configuring_inventory_file.adoc

```diff
@@ -791,6 +791,59 @@ You can use the default *node-config-master-infra* node group to achieve this:
 master.example.com openshift_node_group_name='node-config-master-infra'
 ----
 
+[[configuring-project]]
+== Configuring Project Parameters
+
+To configure the default project settings, configure the following variables in
+the  *_/etc/ansible/hosts_* file:
+
+[[advanced-project-parameters]]
+.Project Parameters
+[options="header"]
+|===
+
+|Parameter |Description |Type |Default Value
+
+|`osm_project_request_message`
+|The string presented to a user if they are unable to request a project via the
+projectrequest API endpoint.
+|String
+| null
+
+|`osm_project_request_template`
+|The template to use for creating projects in response to projectrequest. If you
```

Comment:
Should this be:

> ...in response to **a** projectrequest.

This is my ignorance with 3.x showing. :)

Reply:
I think you're right. I'm fixing this in both instances. Thanks!